### DISCUSSION TITLE
Accessing a discussion board

### DISCUSSION THREAD

**Discussion by MichelleSanF** [community/community#27353] [Discussions]
I have received an invitation from my group to join Github to join a discussion board.  I have tried from two email addresses.  I set up two profiles and user names.  I can sign on, but there is no way it takes me to the discussion board.  It takes me to a search bar for repositories, but when I search my group, it does not come up. In addition, I seem to be the only one having trouble.  perhaps you can assist me, or delete my profiles and I can start over?
Please call or write to me at 201 400 7982  or MichelleSF21@gmail.com

Thanks
MichelleSF

**Comment by radar-blip** [comment]
There are 2 types of discussions. Community Discussions (which is where you are now).
And individual repository discussions, if you and your group are working on a project, you need to open that repository and click on the discussion tab. Maybe there was a misunderstanding about discussions within your group.

**Reply by MichelleSanF** [reply]
How do I find that repository?  They have given me the name, but when I type it in:  NJ-stroke-coordinators....it does not appear for me to find it and go to their discussion board

**Reply by airtower-luna** [reply]
This sounds like there's some step missing here. Is the repository private? If yes you must have accepted the invitation before you can see it. The easiest thing would be if your team could give you a link to the repository, if you get a 404 Not Found error you don't have a access yet.

**Comment by MichelleSanF** [comment]
Yes,  They have sent me several invitations.  It's very strange.
So I have accepted and created my account....which they can see....I just
cannot find my way back and I do get the error message each time they
resend me and invitation........Should we start over?  Can you delete my
account completely?

Thanks for your help
Michelle San Fillipo

**Reply by airtower-luna** [reply]
> So I have accepted and created my account

Did you click on the acceptance link while logged in to your account?

> Should we start over?  Can you delete my account completely?

Mind that this is a public forum, most of the people here are other users, and we can't delete your account. You can [do it yourself it you want](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/deleting-your-personal-account), but it's not going to help. Best figure out how to actually accept that invitation. :wink:
